clevertap-core/src/test/java/com/clevertap/android/sdk/inapp/images/FileResourceProviderTest.kt

```diff
@@ -2,23 +2,32 @@ package com.clevertap.android.sdk.inapp.images
 
 import android.graphics.Bitmap
 import com.clevertap.android.sdk.TestLogger
+import com.clevertap.android.sdk.utils.CTCaches
+import com.clevertap.android.sdk.utils.FileCache
+import com.clevertap.android.sdk.utils.LruCache
 import io.mockk.*
+import org.junit.Before
+import org.junit.Ignore
 import org.junit.Test
+import org.mockito.Mockito
 import java.io.File
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
 
+@Ignore
```

Comment:
Does this need to be unignored before merging?

Reply:
Fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/5305e4c1666d3ef2fac96a4c517e039964a0a861